<ui version="4.0" >
 <class>ICQSearchBase</class>
 <widget class="QWidget" name="ICQSearchBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>408</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item row="0" column="0" >
    <widget class="QTabWidget" name="tabWidget3" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="MinimumExpanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabWhitepages" >
      <attribute name="title" >
       <string>ICQ Whitepages Search</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="textLabel2" >
         <property name="text" >
          <string>&amp;Nickname:</string>
         </property>
         <property name="buddy" >
          <cstring>nickName</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QLineEdit" name="nickName" />
       </item>
       <item row="0" column="2" >
        <widget class="QLabel" name="textLabel3" >
         <property name="text" >
          <string>&amp;First name:</string>
         </property>
         <property name="buddy" >
          <cstring>firstName</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="3" >
        <widget class="QLineEdit" name="firstName" />
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="textLabel6" >
         <property name="text" >
          <string>&amp;Email:</string>
         </property>
         <property name="buddy" >
          <cstring>email</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QLineEdit" name="email" />
       </item>
       <item row="1" column="2" >
        <widget class="QLabel" name="textLabel4" >
         <property name="text" >
          <string>&amp;Last name:</string>
         </property>
         <property name="buddy" >
          <cstring>lastName</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="3" >
        <widget class="QLineEdit" name="lastName" />
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="textLabel8" >
         <property name="text" >
          <string>&amp;Gender:</string>
         </property>
         <property name="buddy" >
          <cstring>gender</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="QComboBox" name="gender" />
       </item>
       <item row="2" column="2" >
        <widget class="QLabel" name="textLabel10" >
         <property name="text" >
          <string>&amp;City:</string>
         </property>
         <property name="buddy" >
          <cstring>city</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="3" >
        <widget class="QLineEdit" name="city" />
       </item>
       <item row="3" column="0" >
        <widget class="QLabel" name="textLabel9" >
         <property name="text" >
          <string>Lan&amp;guage:</string>
         </property>
         <property name="buddy" >
          <cstring>language</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1" >
        <widget class="QComboBox" name="language" />
       </item>
       <item row="3" column="2" >
        <widget class="QLabel" name="textLabel11" >
         <property name="text" >
          <string>C&amp;ountry:</string>
         </property>
         <property name="buddy" >
          <cstring>country</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="3" >
        <widget class="QComboBox" name="country" />
       </item>
       <item row="4" column="0" colspan="2" >
        <widget class="QCheckBox" name="onlyOnline" >
         <property name="text" >
          <string>Only search for online contacts</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2" colspan="2" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>166</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabUIN" >
      <attribute name="title" >
       <string>UIN Search</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="textLabel1" >
         <property name="text" >
          <string>&amp;UIN #:</string>
         </property>
         <property name="buddy" >
          <cstring>uin</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QLineEdit" name="uin" />
       </item>
       <item row="1" column="1" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>41</width>
           <height>91</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>82</width>
       <height>181</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" >
    <widget class="QTreeView" name="searchResults" >
     <property name="whatsThis" >
      <string>This is where the results from your search are displayed. If you double-click a result, the search window will close and pass the UIN of the contact you wish to add back to the Add Contact Wizard. You can only add one contact at a time.</string>
     </property>
     <property name="alternatingRowColors" >
      <bool>true</bool>
     </property>
     <property name="rootIsDecorated" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <layout class="QVBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="searchButton" >
       <property name="whatsThis" >
        <string>Search the ICQ Whitepages with your search criteria</string>
       </property>
       <property name="text" >
        <string>&amp;Search</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearButton" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="whatsThis" >
        <string>Clears both search fields and results</string>
       </property>
       <property name="text" >
        <string>C&amp;lear</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopButton" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="whatsThis" >
        <string>Stops the search</string>
       </property>
       <property name="text" >
        <string>Stop</string>
       </property>
       <property name="stdItem" stdset="0" >
        <number>26</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="userInfoButton" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="whatsThis" >
        <string>Show information about the selected contact</string>
       </property>
       <property name="text" >
        <string>User Info</string>
       </property>
       <property name="shortcut" >
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget3</tabstop>
  <tabstop>nickName</tabstop>
  <tabstop>firstName</tabstop>
  <tabstop>email</tabstop>
  <tabstop>lastName</tabstop>
  <tabstop>gender</tabstop>
  <tabstop>city</tabstop>
  <tabstop>language</tabstop>
  <tabstop>country</tabstop>
  <tabstop>onlyOnline</tabstop>
  <tabstop>uin</tabstop>
  <tabstop>searchButton</tabstop>
  <tabstop>clearButton</tabstop>
  <tabstop>stopButton</tabstop>
  <tabstop>userInfoButton</tabstop>
  <tabstop>searchResults</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>uin</sender>
   <signal>returnPressed()</signal>
   <receiver>searchButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>481</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel" >
     <x>623</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nickName</sender>
   <signal>returnPressed()</signal>
   <receiver>searchButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>127</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel" >
     <x>610</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>firstName</sender>
   <signal>returnPressed()</signal>
   <receiver>searchButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>416</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel" >
     <x>595</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
